README.md

```diff
@@ -1,9 +1,7 @@
-# Welcome to Blogging Category
+<!-- markdownlint-disable-next-line -->
+<div align="center"><p><a href="https://app.netlify.com/sites/devprotocol/deploys"><img src="https://img.shields.io/netlify/bddfebe4-8553-4de6-9ddb-522ce7f67842?style=flat-square&logo=netlify"></a> <a href="https://discord.gg/VwJp4KM"><img src="https://img.shields.io/discord/547215761341546497?style=flat-square&logo=discord"></a></p><br><img height="70px" src="https://github.com/web3community.png"><br><h1>Web3community Blog</h1><br><p><a href="https://dev.to/web3community">Visit blog!</a> · <a href="#">Request a article</a> · <a href="#">Write for us 🍴</a></p></div>
```

Comment:
This is the wrong Netlify link. Please fix it. 🙂

Reply:
```suggestion
<div align="center"><p><a href="https://discord.gg/VwJp4KM"><img src="https://img.shields.io/discord/547215761341546497?style=flat-square&logo=discord"></a></p><br><img height="70px" src="https://github.com/web3community.png"><br><h1>Web3community Blog</h1><br><p><a href="https://dev.to/web3community">Visit blog!</a> · <a href="#">Request a article</a> · <a href="#">Write for us 🍴</a></p></div>
```